spyder_env_manager/spyder/widgets/main_widget.py

```diff
@@ -307,9 +441,9 @@ def _message_update_packages(self):
 
     def _message_import_environment(self):
         title = _("Import Python environment")
-        messages = [_("Manager to use"), _("Packages file")]
-        types = ["ComboBox", "Other"]
-        contents = [{"conda-like"}, {}]
+        messages = [_("Manager to use"), "Environment Name", _("Packages file")]
```

Comment:
```suggestion
        messages = [_("Manager to use"), _("Environment name"), _("Packages file")]
```

It seems that this string should translatable.